etc/p3-config/p3-config.yml

```diff
@@ -223,6 +223,7 @@ p3_admin_roles:
 # List of roles to apply to admin users in the p3 project.
 p3_user_roles:
   - heat_stack_owner
+  - observer
```

Comment:
Hmm, what do we want to allow here?

Observer doesn't appear to be able to read secrets, I may be reading that wrongly.

We might want creator role here?
https://github.com/openstack/barbican/blob/master/etc/barbican/policy.json#L4

Having said that, I am all up for trying the lower level of privilege, and seeing how we go with that.

Reply:
I was basing it on https://docs.openstack.org/barbican/latest/admin/access_control.html, which says:

*observer*
Users with this role are allowed to access to existing resources but are not allowed to upload new secrets or delete existing secrets.

- Looks like it works, but the test case is in Australia and they are finished for the day...

Reply:
Ah, turn out that is correct.

observer -> all_users -> secret_non_private_read -> secret:get

So darn easy to miss a link when reading those policy files.